src/app/public/src/modules/clipboard/clipboard.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, ElementRef, AfterViewInit, Renderer2 } from '@angular/core';
+
+@Component({
+  selector: 'stache-clipboard',
+  templateUrl: './clipboard.component.html',
+  styleUrls: ['./clipboard.component.scss']
+})
+export class StacheClipboardComponent implements AfterViewInit {
+  private defaultTigger = 'clipboardButton';
+  private clipboardButton: HTMLButtonElement;
```

Comment:
You can probably get away with using `ViewChild` instead of referencing the DOM directly.
https://angular.io/api/core/ViewChild